components/02-form-elements/typeahead/README.md

```diff
@@ -0,0 +1,57 @@
+### Typeahead
+A component that allows the user to search and select an option from a long list of options, with the ability to navigate suggested results using the keyboard.
+
+Falls back to a standard select when JavaScript is unavailable.
+
+Allows alternative search values for bilingual usage (e.g. A user with language set to Welsh can still search _United Kingdom_ for _Y Deyrnas Unedig_ when looking for countries, or a user can type _Scotland_ to select _United Kingdom_).
+
+related: [GDS Prototype](https://govuk-location-picker-demo.herokuapp.com/country-of-birth)
+
+### Usage
+The component enhances a select dropdown. It will ignore any options where the value is empty (i.e. Please select an option).
+
+When the JavaScript loads it will hide the select and show a text input. It will also change the for attribute on the label to match the id of the text input. When the user selects a value it will update the value of the select and fire an onchange event on the select. If the user types an exact match for a value (case insensitive) it will automatically select this value.
+
+Alternatives can by added as a coma separated list. When a user selects an option via an alternative the matched alternative will be shown in brackets next to the suggestion, however the main text and value will still be used (i.e. User searches _Wales_ - Suggestion will be _United Kingdom (Wales)_ - Selection will change input value to _United Kingdom_ and select value to _GB_).
```

Comment:
```suggestion
Alternatives can by added as a comma separated list. When a user selects an option via an alternative the matched alternative will be shown in brackets next to the suggestion, however the main text and value will still be used (i.e. User searches _Wales_ - Suggestion will be _United Kingdom (Wales)_ - Selection will change input value to _United Kingdom_ and select value to _GB_).
```

A _coma_ separated list would be a very different thing!

Reply:
Haha i didn't even notice! But yes I expect everyone to enter some form of mild coma whilst separating the list 😉. 
(Will update now)